.github/workflows/website.yaml

```diff
@@ -18,9 +18,7 @@ jobs:
         uses: actions/checkout@v1
 
       - name: Build Website
-        run: |
-          cd website
-          yarn install && yarn build
+        run: bazel build //website:website
 
       - name: Deploy Website 🚀
         uses: JamesIves/github-pages-deploy-action@4.1.0
```

Comment:
Is this deploy action going to work here? I feel like it's probably not going to look in the right spot for the website to deploy.

Reply:
Mmm, I see. I'm not really familiar with the github actions or how yarn does things.

I'll figure out where it needs to run to deploy.

Reply:
I think we might just need to run the bazel build before the yarn build, so that `bazel-bin` gets created. Then the `yarn build` will be able to resolve the paths from `bazel-bin` properly.

Reply:
I changed it to unpack the tar that bazel create in `website/build`, which should fix everything, I think.